.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Fix
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is
+
+**To Reproduce**
+Steps to reproduce the behavior (attach screenshots if applicable):
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**What happened?**
+A clear and concise description of what the behaviour was. 
+
+**What did you expect to happen?**
+A clear and concise description of what you expected to happen.
+
+**Browser Environment (please complete the following information):**
+- OS: [e.g. Plan9, HURD, etc]
```

Comment:
Not sure that the usual user would understand the example!
Maybe Linux, Ubuntu, Windows7, Windows10 would be more effective

Reply:
Oberon would be a great addition